modules/metrics/metrics.go

```diff
@@ -71,14 +75,23 @@ func parseCaddyfile(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler, error)
 
 // UnmarshalCaddyfile sets up the handler from Caddyfile tokens. Syntax:
 //
-//     metrics <matcher>
+//     metrics <matcher> {
+//         disable_openmetrics
+//     }
 //
 func (m *Metrics) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 	for d.Next() {
 		args := d.RemainingArgs()
 		if len(args) > 0 {
 			return d.ArgErr()
 		}
+
+		for d.NextBlock(0) {
+			switch d.Val() {
+			case "disable_openmetrics":
+				m.DisableOpenMetrics = true
+			}
```

Comment:
Add a `default:` to reject unsupported subdirectives pls

Reply:
Also - minor thing but the Go property is called "DisableOpenMetrics", so should the JSON be "disable_open_metrics"? Consistency in case :thinking:

Reply:
If `OpenMetrics` is cased that way only because of branding, then `disable_openmetrics` is probably better. But if it is actually two words, "Open Metrics", then `disable_open_metrics` should be preferred. IMO.

Reply:
Seems like one-word branding: https://openmetrics.io/ so yeah, it's fine I guess. Casing rules can be tricky lol

Reply:
Yeah - the convention seems to be `OpenMetrics` and `openmetrics` depending on context. For example, the MIME type for the text exposition format is `application/openmetrics-text`, not `application/open-metrics-text`.

Reply:
> Add a `default:` to reject unsupported subdirectives pls

Fixed